materials/knitr.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Knitr
+layout: default
+---
+
+## Literate programming
```

Comment:
I used `###` in the lecture materials revision unless there were nested headings. Then, the top level went to `##`. To be consistent, all of these headings could be changed to `###`.